packages/react-components/react-migration-v0-v9/package.json

```diff
@@ -38,7 +38,7 @@
     "@fluentui/react-utilities": "^9.9.2",
     "@fluentui/react-jsx-runtime": "9.0.0-alpha.6",
     "@fluentui/react-components": "^9.21.0",
-    "@fluentui/react-northstar": "^0.66.4",
+    "@fluentui/react-northstar": "^0.66.5",
```

Comment:
this will be done automatically when releasing v9, thus should be removed from this PR. cc @ling1726  to double check/verify